src/applications/mhv/secure-messaging/tests/e2e/pages/PatientInboxPage.js

```diff
@@ -435,7 +435,7 @@ class PatientInboxPage {
     cy.get('#compose-message-body')
       .shadow()
       .find('#textarea')
-      .type('testMessage');
+      .type('testMessage', { force: true });
```

Comment:
@fazilqa @haniasma Looks like there are repetitive blocks of code for this operation that can be consolidated in the future into a page class function perhaps